### ISSUE TITLE
Cannot fetch single pair data

### ISSUE THREAD

**Issue by markuskuusisto** [tradingstrategy-ai/trading-strategy#44]
Version: `trading-strategy==0.8.5`

Cannot fetch data for single pair only, this is an issue for < 1h precision time buckets since size of fetching all candles for those is gigantic.

```
import pandas as pd
from tradingstrategy.client import Client
from tradingstrategy.timebucket import TimeBucket
from tradingstrategy.exchange import ExchangeUniverse
from tradingstrategy.chain import ChainId
from tradingstrategy.pair import PandasPairUniverse

client = Client.create_jupyter_client()
exchange_universe: ExchangeUniverse = client.fetch_exchange_universe()
exchange = exchange_universe.get_by_chain_and_slug(ChainId.ethereum, "uniswap-v3")
pair_data = client.fetch_pair_universe().to_pandas()
pair_universe = PandasPairUniverse(pair_data)

pair = pair_universe.get_one_pair_from_pandas_universe(
    exchange.exchange_id,
    "WETH",
    "DAI",
    pick_by_highest_vol=True,
    )

pair_id = pair.pair_id

all_candles = client.fetch_all_candles(TimeBucket.h4)
candles_dataframe_pair = all_candles.to_pandas()

# THIS WORKS
pair: pd.DataFrame = candles_dataframe_pair.loc[candles_dataframe_pair["pair_id"] == pair_id]

# THIS DOES NOT WORK
candles_dataframe_pair = client.fetch_candles_by_pair_ids({pair_id}, TimeBucket.h4)
```

Throws error:

```
Traceback (most recent call last):
...
    raise NoJSONLData(f"Did not get any data, url:{api_url}, params:{params}")
tradingstrategy.transport.jsonl.NoJSONLData: Did not get any data, url:https://tradingstrategy.ai/api/candles-jsonl, params:{'pair_ids': '2697587', 'time_bucket': '4h'}
```

**Comment by hieuh25** [comment]
@markuskuusisto thank you for your report and detailed instruction on how to produce

My guess is this issue is applicable to Uniswap v3 pairs only. I bumped the effort from S to to M as it might not be trivial to fix.

**Comment by AlexTheLion123** [comment]
@markuskuusisto this issue has been solved by PR #48, which should be released soon

**Comment by markuskuusisto** [comment]
@AlexTheLion123 Great it works.

Slight note, I first upgraded the pip package to `0.9.1` and it throwed error:

```
Traceback (most recent call last):
...
    df = load_trading_strategy_like_jsonl_data(
  File "lib/python3.10/site-packages/tradingstrategy/transport/jsonl.py", line 150, in load_trading_strategy_like_jsonl_data
    translated_key = mappings[key]
KeyError: 'tc'
```

However it started to work when I installed the package directly from source which is 0.9.1 as well. So pip package might be broken.
